using_images/other_images/jenkins.adoc

```diff
@@ -844,7 +844,10 @@ for more information.
 
 The {product-title} Pipeline Plug-in is a prior integration between Jenkins and
 {product-title} which provides less functionality than the {product-title}
-Client Plug-in. It remains available and supported.
+Client Plug-in. It remains available and supported to work with {product-title}
+versions up to v3.11.  For later verions of {product-title}, either use the
+`oc` binary directly from your Jenkins Pipelines, or use the 
+xref:#client-plug-in[{product-title} Client Plug-in].
```

Comment:
I would change "it remains available and supported" to "It has been deprecated but continues to work with..."

Reply:
update pushed